data/formatters/generic.yaml

```diff
@@ -462,6 +462,24 @@ short_message:
 - '{file_md5}'
 ---
 type: 'conditional'
+data_type: 'nids:alert:fast'
```

Comment:
I'll rename this to snort, to disambiguate with other NIDS

Reply:
Okay, the parser can process Suricata's `fast.log` as well, but I'm certainly fine with it since it all goes back to snort. 
Changed in 0b5ab6edd7244e8315933ec6a828679672ac7f38. 

Do you think the module and class names should be renamed as well?

Reply:
IMHO yes, but changes are on the way to also make this a text (parser) plugin